README.md

```diff
@@ -226,6 +226,24 @@ Tips:
 1. For structured facts index into the specific fact value otherwise the entire array will come over as a string and ultimately be difficult to use.
 2. Dynamic facts such as CPU usage, Uptime, and others that are expected to change each run are not ideal for tagging.  Static facts that are expected to stay for the life of a node are best candidates for tagging.
 
+### Logs Setup
+
+To configure logs, first set `logs_enabled: true`, and it can then be configured with a class. An example is below:
```

Comment:
```suggestion
Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file by setting `logs_enabled: true`.

Logs can then be configured with a class, for example:
```

Reply:
Will send another PR to reword. This is going in the puppet configuration, and not the datadog.yaml.